src/main/java/ru/spbstu/service/VariantService.java

```diff
@@ -0,0 +1,30 @@
+package ru.spbstu.service;
+
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+import ru.spbstu.model.Variant;
+import ru.spbstu.reader.dto.VcfRecord;
+import ru.spbstu.repository.VariantRepository;
+
+import java.util.stream.Stream;
+
+@Service
+@RequiredArgsConstructor
+public class VariantService {
```

Comment:
i'd suggest to always create interfaces for services.
maybe we should make methods transactional?